### PR TITLE
docs: update zh Doc Introduction index.md

### PR DESCRIPTION
**Pull request by unlimitedcodeG**
add China Doc introduction explanation

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by unlimitedcodeG** [comment]
If mantainer will accept my PR， I will continue update this doc.

File: packages/appium/docs/zh/index.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 ---
-title: Appium文档
+title: Appium文档首页
 ---
+![image](https://github.com/appium/appium/assets/48639567/9dbaf0a0-df6c-4d61-be3e-617653ee1065)
```
Comment (eglitise) [inline packages/appium/docs/zh/index.md:4]: Not sure if linking to the file through URL is the best approach. Probably better to use the same HTML code from the EN docs (to match the image position and size), just change `src="../en/assets/images/appium-logo-horiz.png"`
In the future we will probably move all images outside the EN directory, so docs for other languages don't need to always check the EN directory.

File: packages/appium/docs/zh/index.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
 
+欢迎来到Appium文档！Appium是一个开源项目及其相关软件生态系统，旨在促进许多应用平台的用户界面自动化，包括移动设备（iOS、Android、Tizen）、浏览器（Chrome、Firefox、Safari）、桌面（macOS、Windows）、电视（Roku、tvOS、Android TV、Samsung）等。
+如果您是第一次接触Appium，建议您从 [介绍](https://github.com/appium/appium) 
+开始，然后进入 [快速入门](https://github.com/appium/appium)。
+您随时可以在 [GitHub](https://github.com/appium/appium) 上找到Appium的代码。
+Appium主要用于软件测试自动化领域，以帮助确定给定应用程序的功能是否按预期工作。与其他类型的软件测试不同，UI自动化允许测试人员编写代码，模拟在应用程序的实际用户界面中执行用户场景，尽可能地模拟真实世界中发生的情况，同时实现自动化的各种好处，包括速度、规模和一致性。
+Appium旨在提供一套工具，以一种标准的方式支持跨任意数量的平台进行这种自动化。大多数平台都提供在某个层次上允许进行UI自动化的工具，但这些通常是特定于平台的，需要专业知识以及特定的编程语言经验和工具链。
+Appium试图将所有这些自动化技术统一到一个稳定的界面下，通过大多数流行的编程语言访问（您可以使用Java、Python、Ruby、JS等编写Appium脚本）。
+要了解Appium如何实现这一目标以及涉及的各种组件，请阅读我们的[介绍](https://appium.io/docs/en/2.2/intro/)。
```
Comment (eglitise) [inline packages/appium/docs/zh/index.md:13]: This and the other `appium.io` links are hardcoded to `2.2`, which is not good. For now just use the filepath to the EN docs (`../en/intro/index.md`)

**Review by eglitise — CHANGES_REQUESTED** [review]
Thank you!
Unfortunately it seems that you were translating from an outdated version :( Could you please update with the contents of the [current docs index file](https://github.com/appium/appium/blob/master/packages/appium/docs/en/index.md?plain=1)?

**Comment by jlipps** [comment]
thanks very much for this! once the issues @eglitise has mentioned are resolved, we can add you to @appium/documentors which we try to alert whenever there's a docs change (so you know when a translation update might be required).

File: packages/appium/docs/zh/index.md (hunk 2 above)
Comment (unlimitedcodeG) [reply packages/appium/docs/zh/index.md:13]: ok, thank you very much, I will correct it.

File: packages/appium/docs/zh/index.md (hunk 1 above)
Comment (unlimitedcodeG) [reply packages/appium/docs/zh/index.md:4]: Get it, I will correct it.

**Comment by unlimitedcodeG** [comment]
All replies may be late due to jet lag issues, but I'll definitely get back to you guys when I see a question!

**Comment by unlimitedcodeG** [comment]
> thanks very much for this! once the issues @eglitise has mentioned are resolved, we can add you to @appium/documentors which we try to alert whenever there's a docs change (so you know when a translation update might be required).
Got it ,gentleman.

**Review by eglitise — CHANGES_REQUESTED** [review]
This is unfortunately translated from the quickstart document, not the main landing page. [The correct file is this one](https://github.com/appium/appium/blob/master/packages/appium/docs/en/index.md?plain=1).

**Comment by unlimitedcodeG** [comment]
> This is unfortunately translated from the quickstart document, not the main landing page. [The correct file is this one](https://github.com/appium/appium/blob/master/packages/appium/docs/en/index.md?plain=1).

thank u,  I will correct it.

**Review by eglitise — APPROVED** [review]
Thank you! The links to other pages won't work for now, but if you are planning to continue this translation, they will just fix themselves :)

**Pull request merged**